src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString() {
+        return getDescription();
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
+    }
+
+    public String getDescription() {
+        return "TASK: " + description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void unmarkAsDone() {
```

Comment:
Consider renaming the method for better clarity
```suggestion
    public void markAsUndone() {
```